vite.config.js

```diff
@@ -9,6 +9,9 @@ export default defineConfig({
       checker: 'vue-tsc',
       ignoreSourceErrors: true,
     },
+    fakeTimers: {
+      enable: true,
+    },
```

Comment:
this should mean any timers in spec files are fake. That's how I personally would like to write tests but I don't want to overreach here if you don't like that